src/main/java/mesosphere/marathon/client/model/v2/UpgradeStrategy.java

```diff
@@ -0,0 +1,31 @@
+package mesosphere.marathon.client.model.v2;
+
+
+import mesosphere.marathon.client.utils.ModelUtils;
+
+public class UpgradeStrategy {
+
+    private double minimumHealthCapacity = 0.5;
```

Comment:
`App.upgradeStrategy` is null by default, which means if someone wants to specify update strategy it'll have to create new `UpdateStrategy` instance and it'll probably put some thoughts  about `minimumHealthCapacity` and `maximumOverCapacity`. Also, all other classes have only default no argument constructor.